Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -3969,6 +4113,61 @@ B3::PatchpointValue* B3IRGenerator::createTailCallPatchpoint(BasicBlock* block,
     return patchpoint;
 }
 
+bool B3IRGenerator::canInline() const
+{
+    unsigned inlineDepth = 0;
+    Checked<uint32_t> inlinedBytes = 0;
+    for (auto* currentFrame = this; currentFrame; currentFrame = currentFrame->m_inlineParent) {
+        ++inlineDepth;
+        inlinedBytes += currentFrame->m_inlinedBytes;
```

Comment:
Isn't it only computing inlinedbytes of this callstack frame? We should compute this globally.